boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -61,6 +64,17 @@ if [[ -z "$IMAGE_DIGEST" ]]; then
 fi
 REPO_DIGEST=${operator_image}@${IMAGE_DIGEST}
 
+# Given a supplementary image is specified,
+# generate the image digest.
+if [ ! -z supplementary_image ]; then
+    IMAGE_DIGEST=$(skopeo inspect docker://${supplementary_image}:v${operator_version} | jq -r .Digest)
```

Comment:
As written we're assuming both images (the primary and the supplementary) get built every time, such that they have matching image tags. I didn't go look at the mechanics of the build process, but it seems like, in almost all cases, one or the other will be identical to the previous version.

[Later] I did go and refresh my memory on the build process. I now remember writing the support for additional images. And I thought through possible alternatives and none seem worth the trouble.

So nothing to act on here, but I'll just express mild sadness at the inefficiency of producing and storing an effectively identical image with every PR.

Reply:
I can resonate with your mild sadness, it would indeed need a partial overhaul of the image building pipeline which I reckon would be a reach to support a simple functionality such as this.